src/styles.css

```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
```

Comment:
Does this need to be included globally or can we move it to the local css files of the components which are using them?

Reply:
Since all the components are using it. I thought of making it global.

Reply:
I can remove it if everyone agrees.

Reply:
Sometimes importing themes in global styles.css throws a lot of naming conflicts with the stuff defined in local css files for your components.
styles.css is meant for adding themes which we want to utilize throughout our website. If you plan on using them only for the toolbars, then it is better to move them to the local css file.